docs/source/pages/examples/index.md

```diff
@@ -0,0 +1,41 @@
+:html_theme.sidebar_secondary.remove:
+
+(examples)=
+# Examples
+
+
+::::{grid} 1 1 2 2
+:gutter: 4
+
+
+:::{grid-item-card} {fas}`desktop;sd-text-primary` Acquisition Script
+:link: acquisition-script
+:link-type: doc
+:class-img-top: tutorial-link-image
+
+Using ``datashuttle`` to create project folders in a behavioural-data acquisition script.
+
+:::
+
+
+:::{grid-item-card} {fas}`desktop;sd-text-primary` Validate A Labs Projects
```

Comment:
The grammar here is wrong (I think it's correct on the page itself)

Reply:
I tried changing this to `Validate all projects in a lab`, as I didn't like the old name anyway, but don't like that much better. Can't quite figure out the correct combination of words.